vendor/CMakeLists.avb.txt

```diff
@@ -0,0 +1 @@
+install(PROGRAMS avb/avbtool.py DESTINATION bin RENAME avbtool)
```

Comment:
It is better to split this PR into 2 separate commits.

1 - fix of mkbootimg
2 - enabling avb tool

Reply:
@anatol Done, see #79 for `avbtool`.

Reply:
Oh I just realized you meant splitting it to 2 commits in this PR, oh well... let me know if you want them both here and I can close the other.

Reply:
Either works for me.

The current condition looks good to me. There is no need to redo it.

@nmeum are you OK with this PR?